_15thnight/__init__.py

```diff
@@ -147,32 +145,33 @@ def dashboard():
         return render_template(
             'dashboard/admin.html',
             form=form,
+            cat_form=AddCategoryForm(),
             form_error=form_error,
             users=User.get_users(),
             alerts=Alert.get_alerts(),
+            categories=Category.all(),
             delete_user_form=DeleteUserForm(),
-            deleted_user=deleted_user
+            deleted_user=deleted_user,
+            deleted_category=deleted_category
         )
     elif current_user.role == 'advocate':
         # Advocate user, show alert form
         form = AlertForm()
         if request.method == 'POST' and form.validate_on_submit():
             alert = Alert(
                 description=form.description.data,
-                other=form.other.data,
-                shelter=form.shelter.data,
-                food=form.food.data,
-                clothes=form.clothes.data,
                 gender=form.gender.data,
                 age=form.age.data,
-                user=current_user
+                user=current_user,
+                categories=Category.get_by_ids(form.categories.data).all(),
```

Comment:
Since this method should always expect to call `.all()`, the `.all()` call should be moved into the return of `get_by_ids`.